src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -65,7 +65,7 @@ private static String fireAt(Request req) {
         int colInt = Integer.parseInt(col);
         currModel.shootAtComputer(rowInt,colInt);
         currModel.shootAtPlayer();
-
+        //currModel.shootAtPlayer_hard();
```

Comment:
This function should be called. So we should figure how to call it using inheritence